cmd/protoc-gen-go-tableau-checker/go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/tableauio/checker/cmd/protoc-gen-go-tableau-checker
+
+go 1.21
```

Comment:
Please change version to `go 1.20`, to keep consistent with https://github.com/tableauio/checker/blob/master/go.mod

Reply:
`go install` not support `replace`. See https://github.com/tableauio/tableau/pull/181

I will close this PR, just release by the main module version.